sysroot_creator/scripts/create_cc_sysroot

```diff
@@ -143,20 +146,29 @@ def build_workspace_sysroot_image(
     }
     log('Building workspace image {} (this might take some time...)'
         .format(image_tag))
-    # FIXME: this gives no output until the whole image is built,
-    # at least implement some spinner
+
     try:
-        workspace_image, ws_image_build_logs = DOCKER_CLIENT.images.build(
+        docker_client = docker.APIClient(base_url='unix://var/run/docker.sock')
+        log_generator = docker_client.build(
             path='.',
             dockerfile=str(workspace_dockerfile_path),
             tag=image_tag,
             buildargs=buildargs,
             quiet=False,
             nocache=docker_args.nocache,
-            network_mode=docker_args.network_mode)
-        print('Docker build output')
-        print_docker_logs(ws_image_build_logs)
-        print()
+            network_mode=docker_args.network_mode,
+            decode=True)
+        with tqdm(log_generator) as t:
+            for chunk in t:
+                try:
+                    line = chunk['stream'].rstrip().lstrip()
+                    if line == '':
+                        continue
+                    t.set_postfix_str(line)
+                except (KeyError, AttributeError) as e:
+                    print("Error reading the Dockerfile image. The following exception"
```

Comment:
Is this reading the `Dockerfile` image?

Reply:
Yes, its also yielding the logs from the build output/